windtools/amrwind/post_processing.py

```diff
@@ -9,14 +9,31 @@
 class ABLStatistics(object):
 
     def __init__(self,fpath,start_date=None,mean_profiles=False):
```

Comment:
Can we have something like `calc_ti=False` as default? I don't want people to think the TI they get from this reader comes from AMR-Wind. I think adding this as `False` and letting the user opt-in, it becomes more evident what is being done. Plus-- backward compatibility ;)